subprojects/docs/src/docs/release/notes.md

```diff
@@ -21,151 +21,32 @@ Switch your build to use Gradle @version@ by updating your wrapper properties:
 
 `./gradlew wrapper --gradle-version=@version@`
 
-<a name="incremental-changes-api"/>
+<a name="incremental-tasks-api"/>
 
-## New API for Incremental Changes
+## New API for Incremental Tasks
 
-The new `org.gradle.work` package provides behavior that allow access to any input files that need to be processed by an incremental work action.
-The following `Checksum` example task calculates a SHA256 on each given input. Inputs may be updated at any time, for example by adding more inputs
-to the `sources` property or by updating the content of any given input that was previously listed.
+With Gradle, it's very simple to implement a task that is skipped when all of its inputs and outputs are up to date (see [Incremental Builds](userguide/more_about_tasks.html#sec:up_to_date_checks)).
+However, there are times when only a few input files have changed since the last execution, and you'd like to avoid reprocessing all of the unchanged inputs.
+Tasks only reprocessing out-of-date input files are called _incremental tasks_.
```

Comment:
This is confusing. If the input files are separate inputs, then you're looking at plain old incremental build.

> When a task has an input that can represent multiple files and the task only processes those files that are out of date, it's called an _incremental task_.

Does this sound better?

Reply:
Changed in https://github.com/gradle/gradle/pull/8952.